components/uaepass-authenticator/src/main/java/org/wso2/carbon/identity/uaepass/authenticator/UAEPassAuthenticatorConstants.java

```diff
@@ -0,0 +1,120 @@
+package org.wso2.carbon.identity.uaepass.authenticator;
```

Comment:
change the package name
```suggestion
package org.wso2.carbon.identity.authenticator.uaepass;
```

Reply:
Fixed with : [8ae529f35c](https://github.com/wso2-extensions/identity-outbound-auth-uaepass/pull/1/commits/8ae529f35ca061af7bacd547e4e227f49811a8d5)